src/services/platform.ts

```diff
@@ -1,8 +1,10 @@
-import { isPlatform } from '@ionic/react'
+import { isPlatform, getPlatforms } from '@ionic/react'
 
 export const isIos = () => isPlatform('ios')
 
 export const isAndroid = () => isPlatform('android')
 
+export const isAndroidAndWeb = () => !getPlatforms().some(r=>["desktop","electron","pwa"].indexOf(r) >= 0);
```

Comment:
Is the function body correctly returning if the platform is `android` ?
- The button should only be shown on `Android` built app (not browser)
- if you want the button can also be displayed on `iOs`
- no browser should display the button